guides/common/modules/con_supported-usage-of-foreman-components.adoc

```diff
@@ -0,0 +1,50 @@
+[id="supported-usage-of-project-components_{context}"]
+= Supported usage of {Project} components
+
+Usage of all {ProjectName} components is supported within the context of {ProjectName} only as described below.
+Third-party usage of any components is not supported.
+
+{RHELServer}:: 
+Each {ProjectName} subscription includes one supported instance of {RHELServer}.
+Reserve this instance solely for the purpose of running  {ProjectName}.
++
+Not supported: Using the operating system included with {Project} to run other daemons, applications, or services within your environment.
+
+SELinux::
+Ensure SELinux is in enforcing or permissive mode.
++
+Not supported: Installation with disabled SELinux.
+
+Foreman::
+You can extend Foreman with plug-ins packaged with {ProjectName}.
+See https://access.redhat.com/articles/1343683[Satellite 6 Component Versions] in Red{nbsp}Hat Knowledgebase for information about supported Foreman plug-ins.
++
+Not supported: Extending Foreman with plug-ins in the {ProjectName} Optional repository.
```

Comment:
is "{ProjectName} Optional" the name of the repo? If so, maybe add some emphasis?

Reply:
Applied.